.github/workflows/reposettings.yaml

```diff
@@ -18,7 +17,7 @@ jobs:
           echo 'REPOSETTINGS_YML<<EOF' >> $GITHUB_ENV
           cat reposettings.yml >> $GITHUB_ENV
           echo 'EOF' >> $GITHUB_ENV
-      - uses: txqueuelen/reposettings@v1
+      - uses: txqueuelen/reposettings@eae1df5886c9b3a818fc2a8a1ca695d6186ead83
```

Comment:
Why is it set to a commit?

Reply:
internal doc compliance.

Reply:
I think it is soon to make this change.

We should create a configuration for renovatebot to update actions accordingly. We still do not know how it behaves with actions pinned to commit and it might not upgrade the actions. I prefer the "insecurity" of a tag (that can change the underlying commit) than sticking it to the commit and not upgrading it at all.

We can discuss during the standup to create a task to investigate this and estimate the effort to do this change (not only here but all the repositories).

Reply:
"Pin actions to a tag only if you trust the creator" 
I think that is the case and the case of the 90% of the actions we use, even if it is a bit blurry

Reply:
as discussed offline, we will keep the tag until a proper bump+sec-linter is in place.